docs/pages/installation.md

```diff
@@ -1,59 +1,177 @@
 ---
 title: Installation
-description: There are many ways to install Foundation, but if you're just getting started, we have a few suggestions.
+description: There are many ways to install Foundation from downloading the compiled CSS, automated Sass installs to popular package managers. Each one serves a different skill level or preferred workflow. If this is all new to you, don’t worry; The video below will give you an overview of each method and help you decide which is best for you.
```

Comment:
This is super long, to the point I think it doesn't look great as it goes above the vid. Can we reduce this and move more of the content below?